### PR TITLE
use quay instead of devshift

### PR DESCRIPTION
**Pull request by riuvshin**
use quay instead of devshift

### CODE REVIEW

**Comment by riuvshin** [comment]
@ibuziuk PTAL

File: cico_build_publish_images.sh [hunk 1]
```diff
@@ -45,13 +45,17 @@ for d in recipes/dockerfiles/*/ ; do
     docker push ${new_tag}
   done
 
-  # Pushing to 'push.registry.devshift.net'
-  docker login -u $DEVSHIFT_USERNAME -p $DEVSHIFT_PASSWORD push.registry.devshift.net
+  # Pushing to 'quay.io'
+  if [ -n "${QUAY_USERNAME}" -a -n "${QUAY_PASSWORD}" ]; then
```
Comment (l0rd) [inline cico_build_publish_images.sh]: How to you get these env variables? For devshift we used to cat jenkins-env (line 11).

Comment (riuvshin) [reply cico_build_publish_images.sh]: these env vars are set in jenkins

Comment (riuvshin) [reply cico_build_publish_images.sh]: ah, I missed to fetch them, nice catch

Comment (riuvshin) [reply cico_build_publish_images.sh]: updated PR

**Pull request merged**